### PR TITLE
temporary code to run HEAD pillows

### PR DESCRIPTION
**Pull request by snopoke**
* creates new checkpoints (`temp_{old_checkpoint}`
* mgmt command to set new checkpoints to change feed HEAD

### CODE REVIEW

File: corehq/ex-submodules/pillowtop/management/commands/ptop_set_checkpoint_to_head.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+from __future__ import print_function
+from django.core.management.base import BaseCommand, LabelCommand
```
Comment (dimagimon) [inline corehq/ex-submodules/pillowtop/management/commands/ptop_set_checkpoint_to_head.py]: F401 'BaseCommand' imported but unused

File: corehq/ex-submodules/pillowtop/management/commands/ptop_set_checkpoint_to_head.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import sys
```
Comment (dimagimon) [inline corehq/ex-submodules/pillowtop/management/commands/ptop_set_checkpoint_to_head.py]: F401 'sys' imported but unused

File: corehq/ex-submodules/pillowtop/management/commands/ptop_set_checkpoint_to_head.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+from pillowtop import get_pillow_by_name, get_all_pillow_configs
```
Comment (dimagimon) [inline corehq/ex-submodules/pillowtop/management/commands/ptop_set_checkpoint_to_head.py]: F401 'get_all_pillow_configs' imported but unused

File: corehq/ex-submodules/pillowtop/checkpoints/manager.py [hunk 4]
```diff
@@ -35,7 +35,7 @@ def reset_checkpoint(checkpoint_id):
 class PillowCheckpoint(object):
 
     def __init__(self, checkpoint_id):
-        self.checkpoint_id = checkpoint_id
+        self.checkpoint_id = 'temp_{}'.format(checkpoint_id)
```
Comment (benrudolph) [inline corehq/ex-submodules/pillowtop/checkpoints/manager.py]: this ensures we don't overwrite the current checkpoint. that makes sense to me. looks good

**Review by benrudolph — APPROVED** [review]

**Pull request closed without merging**